model/Human-GEM.yml

```diff
@@ -305173,6 +305215,119 @@
       - subsystem:
           - "Acyl-CoA hydrolysis"
       - confidence_score: 0
+    - !!omap
+      - id: "MAR20173"
+      - name: ""
+      - metabolites: !!omap
+          - MAM00670m: -1
+          - MAM01596m: 1
+          - MAM02039m: -1
+          - MAM02984m: -1
+          - MAM20077m: 1
+      - lower_bound: 0
+      - upper_bound: 1000
+      - gene_reaction_rule: "ENSG00000105953 or ENSG00000181192"
+      - rxnNotes: ""
+      - rxnFrom: ""
+      - eccodes: "1.2.4.-"
+      - references: "PMID:29752936"
+      - subsystem:
+          - "Lysine metabolism"
+      - confidence_score: 0
+    - !!omap
+      - id: "MAR20174"
+      - name: ""
+      - metabolites: !!omap
+          - MAM00824m: -1
+          - MAM01596m: 1
+          - MAM02039m: -1
+          - MAM02984m: -1
+          - MAM20078m: 1
+      - lower_bound: 0
+      - upper_bound: 1000
+      - gene_reaction_rule: "ENSG00000248098 and ENSG00000083123"
+      - rxnNotes: ""
+      - rxnFrom: ""
+      - eccodes: "1.2.4.4"
+      - references: "PMID:29752936"
+      - subsystem:
+          - "Valine, leucine, and isoleucine metabolism"
+      - confidence_score: 0
+    - !!omap
+      - id: "MAR20175"
+      - name: ""
+      - metabolites: !!omap
+          - MAM00669m: -1
+          - MAM01596m: 1
+          - MAM02039m: -1
+          - MAM02984m: -1
+          - MAM20079m: 1
+      - lower_bound: 0
+      - upper_bound: 1000
+      - gene_reaction_rule: "ENSG00000248098 and ENSG00000083123"
+      - rxnNotes: ""
+      - rxnFrom: ""
+      - eccodes: "1.2.4.4"
+      - references: "PMID:29752936"
+      - subsystem:
+          - "Valine, leucine, and isoleucine metabolism"
+      - confidence_score: 0
+    - !!omap
+      - id: "MAR20176"
+      - name: ""
+      - metabolites: !!omap
+          - MAM01013m: -1
+          - MAM01596m: 1
+          - MAM02039m: -1
+          - MAM02984m: -1
+          - MAM20080m: 1
+      - lower_bound: 0
+      - upper_bound: 1000
+      - gene_reaction_rule: "ENSG00000248098 and ENSG00000083123"
+      - rxnNotes: ""
+      - rxnFrom: ""
+      - eccodes: "1.2.4.4"
+      - references: "PMID:29752936"
+      - subsystem:
+          - "Valine, leucine, and isoleucine metabolism"
+      - confidence_score: 0
+    - !!omap
+      - id: "MAR20177"
+      - name: ""
+      - metabolites: !!omap
+          - MAM00671m: -1
+          - MAM01596m: 1
+          - MAM02039m: -1
+          - MAM02984m: -1
+          - MAM20081m: 1
```

Comment:
The rxnformula for MAR20177 is `MAM00671m + MAM02039m + MAM02984m 	->	MAM01596m + MAM20081m ` not as the one in the PR`1-hydroxy-propyl-thiamine diphosphate + lipoamide (MAM02393m) -> S-propanoyldihydrolipoamide + thiamin-PP (MAM02984m)`

Reply:
ah yep looks like I accidentally used the new formula for `MAR03800`; I've corrected that